QA/PoroDevTest/src/main/java/common/api_setup/api_common/UserSerialization.java

```diff
@@ -0,0 +1,90 @@
+package common.api_setup.api_common;
+
+
+public class UserSerialization {
+
+     String name;
```

Comment:
all these attributes should be private, since there are getters/setters methods

Reply:
Defined as private.